<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowConfig</class>
 <widget class="QDialog" name="WindowConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Telemetry recording: variable selection</string>
   </property>
   <widget class="QListWidget" name="lb_TelSelected">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Sel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TelAdd">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TelRemove">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Avail">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Available:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lb_TelAvailable">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>221</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TelDefault">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset to default</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_MaxTicksInMemory">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>310</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When recording telemetry, how many maximum ticks (frames) to hold in memory before writing to disk. Lower number means more frequent disk writes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max ticks in memory:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Sound:</string>
   </property>
   <widget class="QLineEdit" name="le_FrameCount">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Lower values reduce latency but might introduce audio problems. </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame count:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Channels">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>2 for stereo 1 for mono</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Channels:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample rate:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SampleRate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>this should match the sample rate of your audio files</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>231</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Brake marker options:</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay between beeps:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_DelayBetweenBeeps">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Time delay between beeps in milliseconds.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_LoopLength">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How many refresh frames does your marker audio file (loop) last ? If your marker sound is being cut too soon, increase this. If your marker sound starts looping, decrease this.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Expected latency (ms):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Loop length:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Latency">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Expected latency of your sound card. Integrated sound cards are around 200ms, VR headsets and dedicated sound cards are around 10-15. </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Next lap detection (m):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_NextLapDetection">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Once lap distance is between 0 and this value, next lap is detected. Don't place markers before this distance.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume multiplier:</string>
    </property>
   </widget>
   <widget class="QSlider" name="hs_Volume">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Volume">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_Marker">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Marker WAV file:</string>
    </property>
    <widget class="QLineEdit" name="le_FileNameMarker">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>For example &quot;marker.wav&quot; located in the sounds folder</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_PreMarker">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Pre-Marker WAV file:</string>
    </property>
    <widget class="QLineEdit" name="le_FileNamePreMarker">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>For example &quot;marker.wav&quot; located in the sounds folder</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_ResetAllToDefault">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>550</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset all to default</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
